sys-devel/gcc/gcc-8.3.0.ebuild

```diff
@@ -21,6 +21,10 @@ fi
 src_prepare() {
 	toolchain_src_prepare
 
+	# https://gcc.gnu.org/bugzilla/show_bug.cgi?id=69471
+	# https://bugs.gentoo.org/672946
+	epatch "${FILESDIR}"/gcc-8.3.0-_fix_avx2_pr69471.patch
```

Comment:
this needs to be within the if statement that checks for the elibc_musl.

Reply:
I read this as 
		`cpu_indicator.patch`
		`posix_memalign.patch`
to be applied if building a cross toolchain with musl as a libc.

Reply:
No, || = or.  So either  elibc_musl *or* [[ ${CATEGORY} = cross-*-musl* ]].

Reply:
Well, if this is what you want you should have a look at the gcc-7 ebuilds. All of them have an extra patch outside of that statement, and it's where I've been looking for an example of common practice.